pglookout/pglookout.py

```diff
@@ -86,82 +110,141 @@ def __init__(self, config_path):
         )
         # cluster_monitor doesn't exist at the time of reading the config initially
         self.cluster_monitor.log.setLevel(self.log_level)
-        self.webserver = WebServer(self.config, self.cluster_state, self.cluster_monitor_check_queue)
+        self.webserver: WebServer = WebServer(self.config, self.cluster_state, self.cluster_monitor_check_queue)
+
+        if not self._is_initial_state_valid():
+            self.log.error("Initial state is invalid, exiting.")
+            sys.exit(1)
```

Comment:
Added this check to fail early. Ideally, we wouldn't need that. But there are some tricky states that we want to avoid. Read the method for more info (a few lines below).